From fbca97fa2d9931f3307d1c19c52ec9ab7a4a9d26 Mon Sep 17 00:00:00 2001
From: Sheng Nan <b38800@freescale.com>
Date: Fri, 1 Mar 2013 10:30:02 +0800
Subject: ENGR00252071-3: mxc_v4l2_capture: enable auto detect of ov5642/5640
 in sabresd

- enable the auto detect for parallel ov5642 and ov5640 in sabresd board.

Signed-off-by: Sheng Nan <b38800@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabresd.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabresd.c b/arch/arm/mach-mx6/board-mx6q_sabresd.c
index c5a4f23..8cbf7fb 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabresd.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabresd.c
@@ -792,7 +792,7 @@ static struct i2c_board_info mxc_i2c0_board_info[] __initdata = {
 		I2C_BOARD_INFO("wm89**", 0x1a),
 	},
 	{
-		I2C_BOARD_INFO("ov5642", 0x3c),
+		I2C_BOARD_INFO("ov564x", 0x3c),
 		.platform_data = (void *)&camera_data,
 	},
 	{
-- 
1.7.9.5

